x-pack/plugins/secops/common/graphql/types.ts

```diff
@@ -0,0 +1,162 @@
+/* tslint:disable */
+/*
+     * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+     * or more contributor license agreements. Licensed under the Elastic License;
+     * you may not use this file except in compliance with the Elastic License.
+     */
+import { GraphQLResolveInfo } from 'graphql';
```

Comment:
Ah man! That is awesome. That slight glitch with the indentation up there. I got the same thing on my machine. That looks like something odd with the plugin but still at least we have the license info now.

👍

Reply:
i fixed it